docs/table/table_and_dataframes.rst

```diff
@@ -0,0 +1,95 @@
+addition, the base `~astropy.table.Column` class holds a ``unit`` attribute as
+metadata to allow tracking of the units of the data for applications not using
+|Quantity|.
+
+Pandas does not provide support for units.
+
+Multi-dimensional and Structured Columns
+----------------------------------------
+
+Astronomers deal with images, spectra, and other multi-dimensional data that are
+commonly stored in a table. An example is a source catalog with an image thumbnail and a
+spectrum for each source. Structured columns are less common, but are useful for storing
+vectorized data like an `~astropy.coordinates.EarthLocation` in a table.
+
+Pandas is not able to natively store multi-dimensional or structured columns.
+
+Lossless representation of FITS and VOTable data via metadata
+-------------------------------------------------------------
+
+The `astropy.table` package is designed to provide lossless representation of FITS and
+VOTable data. This means that when you read a FITS or VOTable file into a table and then
+write it back out, the data will be effectively identical. This is made possible by
+robust support for table and column metadata which allows storing and propagating common
+column information such as the unit, description, and format. For VOTable data, more
+esoteric information like the UCD is maintained.
+
+FITS and VOTable are two important file formats in astronomy, but the idea of lossless
+representation of physical data is critical in many other contexts as well, for instance
+when retrieving data from the numerous data catalogs provided by `astroquery
+<https://astroquery.readthedocs.io/en/latest/>`_.
+
+Pandas provides limited support for metadata, but as of late-2024 it is highlighted as
+"experimental" in the documentation.
+
+Time and Coordinates
+--------------------
+
+Time and coordinates are fundamental to astronomy, and astropy provides robust support
+for them with the `~astropy.time.Time` and `~astropy.coordinates.SkyCoord` classes.
+Arrays of times and coordinates can be natively stored in `astropy.table`, meaning that
+the full power of these objects is available when working with them as columns within a
+table.
+
+Pandas supports `timeseries
+<https://pandas.pydata.org/docs/user_guide/timeseries.html>`_ data, but with key
+limitations:
+
+- Leap seconds are not supported. In many circumstances (for instance planning an
+  observation) this limitation is not acceptable.
+- Pandas times are stored with 64-bit precision, which is not sufficient for some
+  astronomical applications. Astropy uses 128-bit precision for time to allow
+  sub-nanosecond precision over the age of the universe.
+- Different :ref:`time scales <time-scale>` common in astronomy (e.g. TAI, UT1) are
+  not supported.
+- :ref:`Time formats <time-format>` used in astronomy such as the FITS time format are
+  not supported.
+
+Pandas does not support coordinate columns.
```

Comment:
this is significant, maybe you want to divide this section into two, one for time and one for coordinates?